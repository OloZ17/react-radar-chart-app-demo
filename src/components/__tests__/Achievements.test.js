import React from 'react';
import { render } from '@testing-library/react';
import { Achievements } from '../index';

describe('Achievements() Component', () => {
  const skills = [
    'Thing 1',
    'Thing 2',
    'Thing 3',
    'Thing 4',
    'Thing 5',
    'Thing 6',
  ];

  const achievements = [1, 2, 3, 4, 5, 6];

  const handleChangeAchievement = jest.fn();
  const resetAchievement = jest.fn();

  it('should take a snapshot', () => {
    const { asFragment } = render(
      <Achievements
        skills={skills}
        achievements={achievements}
        handleChangeAchievement={handleChangeAchievement}
        resetAchievement={resetAchievement}
      />
    );
    expect(asFragment(<Achievements />)).toMatchSnapshot();
  });
});
