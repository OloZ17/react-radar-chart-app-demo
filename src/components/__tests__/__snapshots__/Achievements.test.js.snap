// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Achievements() Component should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-md-6"
  >
    <h3>
      Achievements
    </h3>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 1
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="1"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 2
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="2"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 3
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="3"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 4
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="4"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 5
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="5"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
    <div
      class="form-inline form-group mb-2"
    >
      <label>
        Thing 6
      </label>
      <input
        name="requirement"
        style="padding: 2px; width: 50px; text-align: center; margin-right: 5px; margin-left: 5px;"
        type="number"
        value="6"
      />
      <button
        class="btn btn-warning"
        type="button"
      >
        Reset
      </button>
    </div>
  </div>
</DocumentFragment>
`;
